python/dgl/distributed/dist_graph.py

```diff
@@ -143,15 +143,16 @@ def _copy_data_from_shared_mem(name, shape):
 class AddEdgeAttributeFromKVRequest(rpc.Request):
     """Add edge attribute from kvstore to local GraphBolt partition."""
 
-    def __init__(self, name, kv_names):
+    def __init__(self, name, kv_names, padding=0):
```

Comment:
this is internal function, let's do not specify default value for it, so we could detect if we miss something. this will enforce us to specify it.